package.json

```diff
@@ -40,102 +43,38 @@
 		"discordapp"
 	],
 	"devDependencies": {
-		"@babel/core": "^7.13.16",
-		"@babel/plugin-proposal-decorators": "^7.13.15",
-		"@babel/preset-env": "^7.13.15",
-		"@babel/preset-typescript": "^7.13.0",
-		"@commitlint/cli": "^12.1.1",
-		"@commitlint/config-angular": "^12.1.1",
-		"@types/jest": "^26.0.22",
-		"@typescript-eslint/eslint-plugin": "^4.22.0",
-		"@typescript-eslint/parser": "^4.22.0",
-		"babel-jest": "^26.6.3",
+		"@babel/core": "^7.14.6",
+		"@babel/plugin-proposal-decorators": "^7.14.5",
+		"@babel/preset-env": "^7.14.7",
+		"@babel/preset-typescript": "^7.14.5",
+		"@commitlint/cli": "^12.1.4",
+		"@commitlint/config-angular": "^12.1.4",
+		"@types/jest": "^26.0.23",
+		"@typescript-eslint/eslint-plugin": "^4.28.0",
+		"@typescript-eslint/parser": "^4.28.0",
+		"babel-jest": "^27.0.5",
 		"babel-plugin-const-enum": "^1.0.1",
 		"babel-plugin-transform-typescript-metadata": "^0.3.2",
-		"eslint": "^7.25.0",
-		"eslint-config-marine": "^8.3.2",
-		"eslint-config-prettier": "^8.2.0",
-		"eslint-plugin-import": "^2.22.1",
+		"eslint": "^7.29.0",
+		"eslint-config-marine": "^9.0.6",
+		"eslint-config-prettier": "^8.3.0",
+		"eslint-plugin-import": "^2.23.4",
 		"eslint-plugin-prettier": "^3.4.0",
-		"husky": "^4.3.8",
-		"jest": "^26.6.3",
+		"husky": "^6.0.0",
+		"jest": "^27.0.5",
 		"lerna": "^4.0.0",
-		"lint-staged": "^10.5.4",
-		"nock": "^13.0.11",
-		"prettier": "^2.2.1",
-		"typedoc": "^0.20.36",
+		"lint-staged": "^11.0.0",
+		"nock": "^13.1.0",
+		"prettier": "^2.3.2",
+		"rimraf": "^3.0.2",
+		"typedoc": "^0.21.2",
 		"typedoc-plugin-lerna-packages": "^0.3.1",
```

Comment:
```suggestion
```

Few points here:
1. `typedoc-plugin-lerna-packages` doesn't work with typedoc versions higher than `0.19.x`.
2. _technically_ typedoc added lerna monorepo support in v0.21, but from me trying it so far, it is pretty broken. For one it cannot read the TS source files anymore when using it, which is just dumb.
3. @kyranet has said he'll be writing @discordjs/ts-docgen, which would supersede the use of typedoc anyway. I'd say just remove the dependencies and give Kyra some much needed support and love for the docgen tool (boy do we ever need it for DJS and Sappphire)

Reply:
Yeah this is just a very old remnant when I created this repository, good catch.